frontend/src/app/page.js

```diff
@@ -1,6 +1,22 @@
+import Link from 'next/link';
+import Image from 'next/image';
+import classes from './page.module.css';
+
 export default function Home() {
   return (
-    
-   <div>Access Nri data explorer</div>
+  <>
+    <div className={classes.topBanner}>
+      <div className={classes.logo}>
+        <Image src="/ACCESS-NRI_logo.png" alt="ACCESS-NRI_logo" fill />
+      </div>
+      <div className={`${classes.title} ${classes.dongle}`}>
+        ACCESS-NRI Data Explorer
+      </div>
+    </div>
+    <div className={`${classes.cardContainer} ${classes.dongle}`}>
+      <Link href="/catelog" className={classes.card}>Experiments Catalog</Link>
+      <Link href="/release-provenance" className={classes.card}>Model Build Database</Link>
+    </div>
+  </>
```

Comment:
Can we swap the ordering of these, and add “(experimental prototype)” to the Data Explorer. I would like to indicate it is not necessarily intended to be a final product.

Reply:
Done.